buildroot/packages/postmerkos/libpostmerkos.c

```diff
@@ -0,0 +1,147 @@
+#include "libpostmerkos.h"
+
+#include <stdbool.h>
+#include <stdio.h>
+#include <math.h>
+#include <stdlib.h>
+#include <string.h>
+#include <time.h>
+
+
+// has_poe returns true if the device is PoE-capable, false otherwise
+bool has_poe() {
```

Comment:
The function `pd690xx_pres_count()` from `libpd690xx` already provides an easy way to determine if the device is PoE-capable. But maybe you wanted to avoid linking against `libpd690xx` here?

Reply:
I think I wrote that before `libpd690xx` existed and then just didn't look at it again. I don't have any problem linking against that lib and the current approach feels a bit fragile anyway.

Reply:
A quick test says `pd690xx_pres_count` does the trick -- much nicer than the approach I had :+1: